File: codewar/6-kyu/data_reverse.py
# https://www.codewars.com/kata/569d488d61b812a0f7000015/train/python
import re
# my sol
def data_reverse(data):
    split_by_eight = re.findall(".{8}", "".join([str(item) for item in data]))
    rev = reversed(split_by_eight)
    int_list = "".join(rev)
    return [int(item) for item in int_list]

File: codewar/6-kyu/dataReverse.js
// https://www.codewars.com/kata/569d488d61b812a0f7000015/train/javascript

function dataReverse(data) {
  let splitByEight = data.map(String).join("").match(/.{8}/g, data);
  return splitByEight.reverse().join("").split("").map(Number);
}


let input = [1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 0, 1, 0, 1, 0, 1, 0];

console.log(dataReverse(input))